rotkehlchen/chain/avalanche/manager.py

```diff
@@ -19,6 +20,7 @@
 log = RotkehlchenLogsAdapter(logger)
 
 AVAX_ADDRESS = '0x9debca6ea3af87bf422cea9ac955618ceb56efb4'
+COVALENT_AVAX_ADDRESS = '0xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
```

Comment:
Can you point me to the api change that lead to this. Our test may have failed but perhaps it's a hiccup in their system. This does not look right.

Reply:
Took the response from the test call. the only one with the AVAX symbol (same as the avax address that we were using) had this `0xxx...` thing. Is what I pasted in the description of the PR

Reply:
Yes but why? Can you check their docs to see why this change? Announcements, anything? This makes no sense.